charts/merlin/values.yaml

```diff
@@ -157,13 +147,16 @@ mlflow:
     postgresqlUsername: mlflow
 
 postgresql:
-  enabled: true
   resources:
     requests:
       memory: 2Gi
       cpu: 500m
   persistence:
     size: 100Gi
+  ## Option to use postgresql "internal" or "external" database type, by default internal is chosen
```

Comment:
I thought we agree to have "cloudsql" and "postgresql" as separate type? Why it's becoming "internal" and "external"

Reply:
I think it's still better to use `enabled` : true/false, rather than `internal` vs `external`